lib/src/tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import DxcTooltip from "./Tooltip";
+import ExampleContainer from "../../.storybook/components/ExampleContainer";
+import DxcButton from "../button/Button";
+import { userEvent, within } from "@storybook/test";
+import DxcInset from "../inset/Inset";
+import Title from "../../.storybook/components/Title";
+import DxcFlex from "../flex/Flex";
+
+export default {
+  title: "Tooltip",
+  component: DxcTooltip,
+};
+
+const Tooltip = () => (
```

Comment:
It is useless to have all tooltip visual tests in a single snapshot since it is not possible to show them all at the same time.

Reply:
The problem is that even though I trigger multiple hovers in the play function, only 1 tooltip shows at a time.

On the other hand, creating a story for each scenario seems overkill.

What is your suggestion?

Reply:
In the end it has been decided that, due to not being able to isolate the tooltip, it is needed to have a separate story for each position